ts/components/dialog/debug/components.tsx

```diff
@@ -351,13 +351,20 @@ export const DebugActions = () => {
       <DebugButton
         onClick={async () => {
           const masterPrivKeyHex = await getProMasterKeyHex();
-          console.warn('masterPrivKeyHex', masterPrivKeyHex.slice(0, 64));
+          window?.log?.info('masterPrivKeyHex: ', masterPrivKeyHex.slice(0, 64));
```

Comment:
I am a bit relunctant to add this as a window.log.info.
We definitely don't want this in prod, hence why it was a console.warn (as we have the eslint no-console)